query/src/configs/config_query.rs

```diff
@@ -40,6 +40,10 @@ const QUERY_RPC_TLS_SERVER_KEY: &str = "QUERY_RPC_TLS_SERVER_KEY";
 const QUERY_RPC_TLS_SERVER_ROOT_CA_CERT: &str = "QUERY_RPC_TLS_SERVER_ROOT_CA_CERT";
 const QUERY_RPC_TLS_SERVICE_DOMAIN_NAME: &str = "QUERY_RPC_TLS_SERVICE_DOMAIN_NAME";
 
+const TABLE_ENGINE_CSV_ENABLED: &str = "TABLE_ENGINE_CSV_ENABLED";
```

Comment:
These configs are read-only, we can't set them from the env.

Reply:
ok